website/static/js/passwordForms.js

```diff
@@ -243,15 +289,42 @@ var SignUpViewModel = oop.extend(BaseViewModel, {
 
 /** Wrapper classes */
 var ChangePassword = function(selector) {
-    $osf.applyBindings(new ChangePasswordViewModel(), selector);
+    ChangePasswordViewModel = new ChangePasswordViewModel();
```

Comment:
This variable needs to be declared. Also, because it's an instance of `ChangePasswordViewModel`, the variable should be lowercased. `viewModel` would be a fine name.